catalog/nsili/catalog-nsili-source/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -20,18 +20,18 @@
         <AD description="The ID of the NSILI Source" name="NSILI Source ID" id="id"
             required="true" type="String" default="NSILI Federated Source"/>
 
-        <AD description="The URL of the IOR File to use for the NSILI CORBA Service.  For local files, use 'file://'"
+        <AD description="The URL of the IOR File to use for the NSILI CORBA Service.  For local files, use 'file://'."
```

Comment:
We may want to list all the protocols that we can support in the url:  http://, https:// , ftp:// , fie://